tools/update-rhcos-image

```diff
@@ -2,9 +2,9 @@
 
 dest="${1:-/var/lib}"
 
-build=$(curl -s https://releases-rhcos.svc.ci.openshift.org/storage/releases/maipo/builds.json | jq -r '.builds[0]')
-image=$(curl -s https://releases-rhcos.svc.ci.openshift.org/storage/releases/maipo/$build/meta.json | jq -r '.images["qemu"].path')
-url="https://releases-rhcos.svc.ci.openshift.org/storage/releases/maipo/$build/$image"
+build=$(curl -s https://releases-art-rhcos.svc.ci.openshift.org/storage/releases/maipo/builds.json | jq -r '.builds[0]')
+image=$(curl -s https://releases-art-rhcos.svc.ci.openshift.org/storage/releases/maipo/$build/meta.json | jq -r '.images["qemu"].path')
+url="https://releases-art-rhcos.svc.ci.openshift.org/storage/releases/maipo/$build/$image"
```

Comment:
I dont' think this file is now required, we can remove it all together.

Reply:
great, ok